lib/manageiq/cross_repo/runner.rb

```diff
@@ -5,20 +5,22 @@ module ManageIQ::CrossRepo
   class Runner
     attr_reader :test_repo, :core_repo, :gem_repos
 
-    def initialize(test_repo, core_repo, gem_repos)
+    def initialize(test_repo, repos)
       @test_repo = Repository.new(test_repo || "ManageIQ/manageiq@master")
+
+      core_repos, @gem_repos = Array(repos).collect { |repo| Repository.new(repo) }.partition(&:core?)
+      @core_repo = core_repos.first
+
       if @test_repo.core?
-        raise ArgumentError, "You cannot pass core repo when running a core test"           if core_repo.present? && core_repo != test_repo
-        raise ArgumentError, "You must pass at least one gem repo when running a core test" if gem_repos.blank?
+        raise ArgumentError, "You cannot pass core repo when running a core test"           if @core_repo.present? && @core_repo != @test_repo
+        raise ArgumentError, "You must pass at least one gem repo when running a core test" if @gem_repos.blank?
```

Comment:
Separate but I think we should get rid of this check...if someone wants to run a core test, why not let them.  We could use this as the "default" in manageiq-cross_repo to allow it to build a bundler cache.

Reply:
:+1: removed